hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OmTableInsightTask.java

```diff
@@ -376,11 +376,36 @@ private void handleUpdateEvent(OMDBUpdateEvent<String, Object> event,
     String countKey = getTableCountKeyFromTable(tableName);
     String unReplicatedSizeKey = getUnReplicatedSizeKeyFromTable(tableName);
     String replicatedSizeKey = getReplicatedSizeKeyFromTable(tableName);
```

Comment:
`countKey, unReplicatedSizeKey, replicatedSizeKey` are not used in this method. No need to pass many arguments in handleSizeRelatedTableUpdateEvent, instead read in `handleSizeRelatedTableUpdateEvent` method itself by passing `tableName`. This will reduce number of arguments passed.

Reply:
Implemented for `handleSizeRelatedTableUpdateEvent`, `handleSizeRelatedTablePutEvent`, `handleSizeRelatedTableDeleteEvent`

Reply:
Thanks for updating, Looks good now.